app/controllers/api/v0/base_controller.rb

```diff
@@ -2,44 +2,43 @@ module Api
   module V0
     class BaseController < ApplicationController
       def list_order_item
-        item = OrderItem.where('id = ? and order_id = ?',
-                               params['order_item_id'], params['order_id']).first
-        render json: item.to_hash
+        render json: Order.find(params.require(:order_id)).
```

Comment:
Is the `Order.find` necessary if we have the `order_item_id`?

Reply:
@mkanoor Do you always need the order_id to be able to access the item?  Is this for security or RBAC?

Reply:
@gmcculloug Its for security and RBAC. So that if only orders have the RBAC to get to the order_item you would need access to the order